pkgs/tools/networking/sshping/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchFromGitHub, libssh}:
+
+stdenv.mkDerivation rec {
+  pname = "sshping";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "spook";
+    repo = "sshping";
+    rev = "v${version}";
+    sha256 = "0p1fvpgrsy44yvj44xp9k9nf6z1fh0sqcjvy75pcb9f5icgms815";
+  };
+
+  buildInputs = [ libssh ];
+
+  buildPhase = ''
+      g++ -Wall -I ext/ -o bin/sshping src/sshping.cxx -lssh
```

Comment:
Why not to use the upstream's Makefile?

Reply:
The upstream Makefile has a hardcoded path for libssh.h (/usr/include/libssh/libssh.h). Since the build a single command, I think it's slightly simpler to have it here than it would be to fix the Makefile.